### ISSUE TITLE
Missing some selectors on Vyper 0.3.10 contracts

### ISSUE THREAD

**Issue by ngotchac** [cdump/evmole#7]
In these curve contracts, not all methods have been found:
- https://etherscan.io/address/0xef484de8C07B6e2d732A92B5F78e81B38f99f95E#code
- https://etherscan.io/address/0x9e10f9Fb6F0D32B350CEe2618662243d4f24C64a#code

Tried with:
```
cast selectors $(cast code 0xef484de8C07B6e2d732A92B5F78e81B38f99f95E) --resolve
```
```
0x081579a5	uint256,int128,uint256,uint256       	remove_liquidity_one_coin(uint256,int128,uint256,address)
0x0b4c7e4d	                                     	add_liquidity(uint256[2],uint256)
0x18160ddd	                                     	totalSupply()
0x1be913a5	                                     	ma_exp_time()
0x23b872dd	uint256,uint256,uint256              	transferFrom(address,address,uint256)
0x30c54085	                                     	withdraw_admin_fees()
0x3644e515	                                     	DOMAIN_SEPARATOR()
0x52d2cfdd	uint256,uint256,uint256,uint256      	remove_liquidity_imbalance(uint256[2],uint256,address)
0xb4b577ad	                                     	future_A()
0xddc1f59d	int128,int128,uint256,uint256,uint256	exchange(int128,int128,uint256,uint256,address)
```

The ABI shows 68 methods (read + write)

**Comment by cdump** [comment]
Confirmed.
In this contract execution flow depends on `CALLDATA` `AND_OPERATOR` `CONSTANT`.
I need time to investigate and think about possible fixes.

**Comment by cdump** [comment]
Fixed in master, will be in the next release.
Vyper does `signature % n_buckets`, but sometimes it's optimized(?) to `signature AND (n_buckets-1)` (with different bucket content of course). Now it's supported.

**Comment by ngotchac** [comment]
That's awesome, great work! Closing this issue then :)

**Comment by cdump** [comment]
Fixed in v0.3.8 release - https://cdump.github.io/evmole/#0xef484de8C07B6e2d732A92B5F78e81B38f99f95E/eth (Ctrl+F5)
